src/ZoneLoading/Game/IW4/ZoneLoaderFactoryIW4.cpp

```diff
@@ -106,7 +106,8 @@ namespace
                 };
             }
         }
-        else if (endianness::FromBigEndian(header.m_version) == ZoneConstants::ZONE_VERSION_XENON)
+        else if (endianness::FromBigEndian(header.m_version) == ZoneConstants::ZONE_VERSION_XENON
+                 || endianness::FromBigEndian(header.m_version) == ZoneConstants::ZONE_VERSION_XENON_ALPHA)
```

Comment:
Depending on the extent of the future features for xenon zones we may need another identifier for a particular build?

Reply:
possible, for now it should do though i guess?